bin/check-typescript-strict

```diff
@@ -3,21 +3,17 @@
 echo "Refreshing Kea logic types and running the TypeScript compiler in dry mode..."
 echo
 
-IGNORED_SPECS=('Type.ts') # *Type.ts* files are ignored, as they are generally created by kea-typegen
```

Comment:
Considering that really the only purpose of this script was making it possible to ignore certain errors, can we now just remove it and update the `typescript:check` script to be `yarn typegen:write && tsc --strict`? Would make the workflow a bit simpler to grasp